packages/oc-template-handlebars/lib/render.js

```diff
@@ -1,14 +1,29 @@
 const handlebars = require('handlebars');
+const Cache = require('nice-cache');
 const utils = require('./utils');
 
+const cache = new Cache();
+const cacheType = 'handlebars-precompiled-template';
```

Comment:
maybe we could stick to the template name, so that in the future other template need a similar strategy they can follow the approach. `oc-template-handlebars-precompiled` , what you think?